crates/milli/src/search/new/vector_sort.rs

```diff
@@ -83,7 +83,7 @@ impl<'ctx, Q: RankingRuleQueryTrait> RankingRule<'ctx, Q> for VectorSort<Q> {
     }
 
     #[allow(clippy::only_used_in_recursion)]
-    #[tracing::instrument(level = "trace", skip_all, target = "search::vector_sort")]
+    #[tracing::instrument(level = "debug", skip_all, target = "search::vector_sort")]
```

Comment:
this might cause a lot of debug logs. logging `fill_buffer` would also be clearer than having both start iteration and next_bucket.

But fill_buffer might still be many logs.

For instance, if you have a sort rule and then the vector rule, you might have one debug log per returned document.